SPECS/glibc/glibc.spec

```diff
@@ -281,21 +278,21 @@ grep "^FAIL: nptl/tst-eintr1" tests.sum >/dev/null && n=$((n+1)) ||:
 
 %files iconv
 %defattr(-,root,root)
-%{_lib64dir}/gconv/*
+%{_libdir}/gconv/*
 
 %files tools
 %defattr(-,root,root)
 %{_bindir}/mtrace
 %{_bindir}/pcprofiledump
 %{_bindir}/xtrace
-/sbin/sln
-%{_lib64dir}/audit/*
-/lib64/libpcprofile.so
+%{_sbindir}/sln
+%{_libdir}/audit/*
+#/lib64/libpcprofile.so
```

Comment:
Not sure if yo uneed to followup on this commented out item.

Reply:
I've filed a bug to track this: https://microsoft.visualstudio.com/OS/_workitems/edit/47485967